openedx/core/djangoapps/course_groups/cohorts.py

```diff
@@ -436,3 +436,48 @@ def is_default_cohort(user_group):
     )
 
     return len(random_cohorts) == 1 and random_cohorts[0].name == user_group.name
+
+
+def set_course_cohort_settings(course_key, **kwargs):
+    """
+    Set cohort settings for a course.
+
+    Arguments:
+        course_key: CourseKey
+        is_cohorted (bool): If the course should be cohorted.
+        always_cohort_inline_discussions (bool): If inline discussions should always be cohorted.
+        cohorted_discussions (list): List of discussion ids.
+
+    Returns:
+        A CourseCohortSettings object.
+
+    Raises:
+        ValueError if course_key is invalid.
```

Comment:
Are there any errors thrown for invalid values, e.g. cohorted_discussions is not a list, or contains non ids?